Makefile

```diff
@@ -31,6 +31,17 @@ npm-ci: node_modules ## Run npm ci
 npm-install: ## Run npm install
 	$(call npm, --workspaces install)
 
+.PHONY: npm-version
+npm-version: ## Run npm install
+ifndef VERSION
+$(error VERSION environment variable is not set)
+endif
+	$(call npm, --workspaces version $(VERSION))
```

Comment:
is the only benefit of this makefile now the nvm version? Seems it would be a lot easier to just run the `npm` command than to wrap it with makefile...

Reply:
So the whole point is that we get a reproducible build and that node and npm versions are correct. How can you guarantee on some random developers machine which versions of node is executed?